.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: I have a feature request for SQLx
+description: Create a feature-request issue
+labels: [enhancement]
+body:
+  - type: textarea
+    id: related-issues
+    validations:
+      required: true
+    attributes:
+      label: I have found these related issues/pull requests
+      description: "I have searched by clicking [HERE](https://github.com/launchbadge/sqlx/issues?q=) for existing issues, these are the ones I've found, and this is why I think this deserves a new issue."
+      placeholder: "Related to #1 by also touching the ... system. They should not be merged because ..."
+  - type: textarea
+    id: feature-description
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description: A clear and concise description of what the problem is
+      placeholder: You should add ...
+  - type: textarea
+    id: solution
+    validations:
+      required: true
+    attributes:
+      label: Prefered solution
+      description: A clear and concise description of what you want to happen.
+      placeholder: In my use-case, ...
+  - type: textarea
+    id: alternatives
+    validations:
+      required: false
+    attributes:
+      label: Alternative solutions
+      description: A clear and concise description of any alternative solutions or features you've considered
+      placeholder: I have considered ...
+  - type: textarea
+    id: additional-context
+    validations:
+      required: false
+    attributes:
+      label: Additional Context
+      description: Add any other context or screenshots about the feature request here
+      placeholder: ...
```

Comment:
What maybe would be nice is a field where they list some relevant areas (e.g. mysql, postgres, macros, etc).

Also, "is this a breaking change? why or why not?"

Reply:
unsure about the wording on this one. We maybe need to iterate on the wording a bit.

What kind of answers would you like to this question?

Reply:
And should this field be mandatory?